examples/Rakefile

```diff
@@ -1,41 +1,111 @@
 require 'fastly_nsq'
 require 'fastly_nsq/rake_task'
 
-##
-# Both topic and channel will need to be passed into the rake task when
-# it is called.
+#-------------------------------------------------------------------------------
+# You are required to to set the topics in MessageProcessor.topics as an array:
 #
-# ex: `rake listen_task[my_topic,my_channel]`
-# topic will be 'my_topic' and channel will be 'my_channel'
+# class MessageProcessor
+#   def self.topics
+#     ['customer_created', 'customer_deleted']
+#   end
 #
-MessageQueue::RakeTask.new(:listen_task, [:topic, :channel])
-
-##
-# In Rails, you can include the environment like so:
-#
-MessageQueue::RakeTask.new(:listen_task, [:topic, :channel] => :environment)
-
-##
-# Topic and channel can also be preset in the task by passing a block and
-# assigning value like so. This is useful if you want a single purpose
-# listener task.
+#   # ...
+# end
 #
-# ex: `rake listen_task`
-# topic will be 'some_topic' and channel will be 'some_channel'
+# A channel will need to be passed in
+# to the Rake task
+# when it is called.
 #
-MessageQueue::RakeTask.new(:listen_task) do |task|
-  task.topic   = 'some_topic'
-  task.channel = 'some_channel'
+# The task looks like:
+
+namespace :queue do
```

Comment:
We certainly don't need it in this example file. Removed in f53aebc.

It is hardwired into the listening cookbooks, so those will need updating if we change in the Rails apps, but they need it anyhow so it would be a good time to adjust. :+1: